crates/core/tedge_actors/src/examples/mod.rs

```diff
@@ -75,8 +73,15 @@
 //! let mut player_2_box = player_2_box_builder.build();
 //!
 //! // Then spawn the server
-//! let server = Calculator::default();
-//! tokio::spawn(ServerActor::new(server).run(server_box));
+//! // TODO: Speak to Didier
+//! //   I've moved the message box from the actor trait and the run method no longer takes a message box  
+//! //   the Calculator impls actor so I've added a new that takes a message box but it also impls server
+//! //   using server means the Calculator can handle requests without having a message box
+//! //   which is a problem because if you want to use Calculator as a server then you shouldn't need to have a message box to construct the Calculator
+//! //   but if you construct without a message box then using calculator by calling Actor::run wont work so what should be done?
```

Comment:
You are correct: it makes no sense for a struct to `impl` both `Actor` and `Server`. This example must be split in  two.

Reply:
Do you want me to do this in this PR?